Lib/fontTools/subset/cff.py

```diff
@@ -401,9 +418,24 @@ def op_callgsubr(self, index):
 		psCharStrings.SimpleT2Decompiler.op_callgsubr(self, index)
 		self.processSubr(index, subr)
 
+	def stop_hint_count(self, index=None):
+		self.need_hintcount = False
+		for op_name in stop_hintcount_ops:
+			setattr(self, op_name, None)
+		cs = self.callingStack[-1]
+		if hasattr(cs, '_desubroutinized'):
+			raise StopHintCountEvent()
+
+	def op_hintmask(self, index):
+		psCharStrings.SimpleT2Decompiler.op_hintmask(self, index)
+		if self.need_hintcount:
+			self.stop_hint_count()
+
 	def processSubr(self, index, subr):
 		cs = self.callingStack[-1]
-		cs._patches.append((index, subr._desubroutinized))
+		if not hasattr(cs, '_desubroutinized'):
+			cs = self.callingStack[-1]
```

Comment:
this line 437 is repeated two lines above

Reply:
Right.